### PR TITLE
Fix: create new genesis for holesky

### PR DESCRIPTION
**Pull request by barnabasbusa**

### CODE REVIEW

**Review by rolfyone — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -12,40 +12,18 @@ This is _work in progress_. The final config will be shared prior to launch date
 * Status: _Planned_ ([#3](https://github.com/eth-clients/holesky/issues/3))
 * Flag: `--holesky`
 * Flavor: Permissionless (Proof-of-Stake), _to replace Goerli_
-* Launch Date: **Sept/15, 2023, 14:00 UTC**
-  * Epoch time: 1694786400
+* Launch Date: **Sept/28, 2023, 12:00 UTC**
```
Comment (michaelsproul) [inline README.md:15]: This time suits the Australians & Kiwis a lot better than 14:00UTC. And Thursday is better than Friday as it gives a bit of a buffer to fix things before the weekend :+1:

File: custom_config_data/config.yaml [hunk 2]
```diff
@@ -6,9 +6,10 @@ CONFIG_NAME: holesky
 # ---------------------------------------------------------------
 # `2**14` (= 16,384)
 MIN_GENESIS_ACTIVE_VALIDATOR_COUNT: 16384
-# Sep-15-2023 13:55:00 +UTC
-MIN_GENESIS_TIME: 1694786100
-GENESIS_FORK_VERSION: 0x00017000
+# Sep-28-2023 11:55:00 +UTC
+# This is an invalid valid and should be updated when you create the genesis
+MIN_GENESIS_TIME: 1695902100
+GENESIS_FORK_VERSION: 0x01017000
```
Comment (hwwhww) [inline custom_config_data/config.yaml:12]: 😱😱😱
then we will have to update deposit-cli for it.

@barnabasbusa 
Is there also a new `genesis_validators_root `?

/cc @carlbeek

**Review by prestonvanloon — APPROVED** [review]
LGTM -- Prysm is no longer complaining about missing genesis block and appears ready to launch holesky with the configs from this PR.

**Comment by rolfyone** [comment]
can we merge and start releasing?

File: custom_config_data/config.yaml (hunk 2 above)
Comment (parithosh) [reply custom_config_data/config.yaml:12]: The `genesis_validators_root` is `0x9143aa7c615a7f7115e2b6aac319c03529df8242ae705fba9df39b79c59fa8b1`

**Review by siladu — COMMENTED** [review]
Needs to incorporate these besu config issues first https://github.com/eth-clients/holesky/pull/72.

Couple of questions I want to clarify, maybe I missed some discussion about this:
1. The new genesis epoch time is 1695902400 but the EL genesis timestamp is 1695902100 (5 mins before). Just wondering why we need that 5 minutes?

2. Why can't we start with shanghai/capella at genesis?

**Comment by parithosh** [comment]
> Besu needs to incorporate #72 first.
> 
> Couple of questions I want to clarify, maybe I missed some discussion about this:
> 
>     1. The new genesis epoch time is 1695902400 but the EL genesis timestamp is 1695902100 (5 mins before). Just wondering why we need that 5 minutes?
> 
>     2. Why can't we start with shanghai/capella at genesis?

Yup, waiting on Barnabas to merge in changes from #72 , context:  https://github.com/eth-clients/holesky/pull/72#issuecomment-1724196281

1. The tool just auto sets the CL config's `MIN_GENESIS_TIME` as the EL genesis timestamp. We can change that to `MIN_GENESIS_TIME` + `GENESIS_DELAY` which is when the CL will have genesis, but it shouldn't play a role as the EL is just basically ready for duties and waits around until the CL sends it the required messages - Would loosely prefer not changing the value for now unless there's a reason to do so
2. We haven't tested capella genesis in any devnet yet(devnet-9 will be the first), we only have tested it in some local testnets and weren't comfortable with testing it publically for the first time at such a huge scale. That's why we opted to suggest bellatrix genesis and a capella fork a day later.

**Comment by siladu** [comment]
Can any other EL teams confirm this is the ForkId for Holesky at genesis?

`ForkId(0xc61a6098, 1696000704)`
and then after Shanghai...
`ForkId(0xfd4f016b, 0)`

**Review by siladu — APPROVED** [review]
LGTM

**Comment by g11tech** [comment]
> Can any other EL teams confirm this is the ForkId for Holesky at genesis?
> 
> `ForkId(0xc61a6098, 1696000704)` and then after Shanghai... `ForkId(0xfd4f016b, 0)`

yes i can confim the same :slightly_smiling_face: 
 - https://github.com/ethereumjs/ethereumjs-monorepo/pull/3049

**Pull request merged**